README.md

```diff
@@ -17,12 +17,14 @@ The packages will aim to support all current .NET versions. Refer to the current
 ### Basic Usage
 
 ```csharp
-using OpenFeature;
+using OpenFeature.Model;

```

Comment:
Needed for EvaluationContext.

Removed OpenFeature, as we use them explicitly. `OpenFeature.Api...`